packages/apollo/src/Command.ts

```diff
@@ -214,10 +214,14 @@ export abstract class ProjectCommand extends Command {
   }
 
   async runTasks<Result>(
-    generateTasks: (context: ProjectContext) => ListrTask[]
+    generateTasks: (context: ProjectContext) => ListrTask[],
+    options?: Listr.ListrOptions | ((ctx: ProjectContext) => Listr.ListrOptions)
   ): Promise<Result> {
     const tasks = await generateTasks(this.ctx!);
```

Comment:
We could include a null check for `this.ctx` with an error that mentions the need to call `init` before any other function

Reply:
That would be great because I didn't know that was necessary.

Reply:
Also a small note: this line including `ctx!` wasn't from this PR. I updated it anyway :)